### ISSUE TITLE
Add support for multiple strings in "elementFilter"

### ISSUE THREAD

**Issue by j-meds** [johnpierson/RhythmForDynamo#49]
Summary also needs updating as it says it supports "fuzzy searching". For example passing in a list or comma delimited string.

```
/// <summary>
/// Provides element filtering options by name. For the filter method, we are using something called "LevenshteinDistance". 
/// This was introduced to me here, http://dynamobim.org/fuzzy-string-matching/.
/// </summary>
```

**Comment by johnpierson** [comment]
The node itself does not do fuzzy searching. That is just how I find the filter method:

So if a user types in `StratsWith`, I can use Fuzzy Search to find the closest match which would be `StartsWith`

**Comment by j-meds** [comment]
@johnpierson Possible to support multiple strings in the value ? For example I need to get elements that contain "Door" and "Window".

**Comment by johnpierson** [comment]
That _might_ be possible. I will have to take a look when I have a chance.

**Comment by j-meds** [comment]
@johnpierson Okay please do let me know, if you don't have enough time I can create a PR for it as well. I believe the changes would be in this file ? https://github.com/johnpierson/RhythmForDynamo/blob/master/src/Rhythm/Revit/ElementFilter/ElementFilter.cs

**Comment by johnpierson** [comment]
Yep! That is where the change would be. The biggest thing to consider (and probably the majority of the time) would be for it to work with all list structures.

---

That being said, it is already doable with list management. (Also if you are looking to filter by category, then the ElementFilter.ByName node is not the correct one to use. But if those values were in the name. then cool)
![image](https://user-images.githubusercontent.com/15744724/180077332-cd661d44-e79a-41c2-9e4f-1689f921822f.png)

**Comment by j-meds** [comment]
updated images

**Comment by johnpierson** [comment]
What does the list structure that you are putting in the elements input look like? That can cause hiccups if it is nested.

**Comment by j-meds** [comment]
@johnpierson I'm opening 3 files and fetching their schedules. I get a list of 3 element list, 
![image](https://user-images.githubusercontent.com/12689117/180081115-ccdefce1-70a0-4462-b27f-2fa993398aa8.png)

I don't want to flatten because I need to keep separate, it's odd it works fine with the string.

**Comment by johnpierson** [comment]
> it's odd it works fine with the string.
Not really. because that is a single item. The other demonstration is a list. Very different when it comes to how dynamo handles it.

If you want this fixed right away, I suggest getting the name yourself with Element.Name and use Clockwork's node called String.ContainsMultiple.

I will try to check it out more in the future, but as of right now, I don't have a lot of time to investigate.

**Comment by j-meds** [comment]
@johnpierson I really appreciate you helping me, It seems that clockwork node doesn't exist anymore and their regex doesn't work with a list that contains multiple list. I went ahead and implemented a python script that merges 2d list as I'm only searching for two keywords. Anyways it would be awesome for multiple strings by using a comma delimited string.

```
list_1 = IN[0]
list_2 = IN[0]

# Place your code below this line
def Merge2dList(lst1,lst2):
	MergedList=[]
	for index,val in enumerate(lst1):
		print(lst1[index])
		MergedList.append(lst1[index] + lst2[index])
	return MergedList

# Assign your output to the OUT variable.
OUT=Merge2dList(IN[0],IN[1])
```
